### PR TITLE
Adm 413  BuildKite integration-Backend

### PR DESCRIPTION
**Pull request by leeranzhi**
## Summary
- [running build backend when only main branch.](https://github.com/au-heartbeat/Heartbeat/commit/8e5503810a706a19e4d68ad8a1803c7c5162b543)

### CODE REVIEW

File: .buildkite/ops/build [hunk 1]
```diff
@@ -11,16 +11,31 @@ display_help() {
     exit 1
 }
 
-build_backend(){
-  echo "build_backend"
+build_backend() {
+    aws configure set region $AWS_REGION
+    eval $(aws sts assume-role --role-arn $AWS_GITHUB_ACTION_ROLE --role-session-name buildkite | jq -r '.Credentials | to_entries | map(.key + "=" + "\"" + (.value | tostring) + "\"") | .[]')
+    aws ecr get-login-password --region $AWS_REGION | docker login --username AWS --password-stdin $AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com
+    set +e
```
Comment (guzhongren) [inline .buildkite/ops/build:18]: Why add this line, same as bellow

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -1,20 +1,27 @@
 env:
   RETENTION_DAYS: "10"
 steps:
-  - label: ":lock: Security Check"
-    command: ./.buildkite/ops/check security
+  # - label: ":lock: Security Check"
```
Comment (guzhongren) [inline .buildkite/pipeline.yml]: Why comment existing code

**Review by guzhongren — COMMENTED** [review]
Need change

File: infra/cloudformation.yml [hunk 3]
```diff
@@ -398,6 +398,16 @@ Resources:
             - sudo yum install nodejs -y
             - sudo npm install -g pnpm
             - sudo yum install java-17-amazon-corretto -y
+            - cat << EOF | sudo tee -a /etc/yum.repos.d/trivy.repo
```
[the remark this answers is not in the document]
Comment (leeranzhi) [reply infra/cloudformation.yml]: fixed.

**Review by guzhongren — APPROVED** [review]

**Pull request merged**